vcpkg.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg-tool/main/docs/vcpkg-schema-definitions.schema.json",
+  "name": "simple-itk",
```

Comment:
We generally go by "SimpleITK" or "simpleitk"., without a hyphen.

Reply:
Fixed